### ISSUE TITLE
different frequency for bass shakers

### ISSUE THREAD

**Issue by nagler2** [SHWotever/SimHub#628]
Hi, I have 2 bass shakers and they are from the same manufacturer but they work different with the same frequency. When I set for example suspension there is only the option to set same frequency for front and rear. I would like to ask you, if it possible in the effects profile to set frequency for each shaker(at least front and rear)? Maybe in the next SimHub versions?
Many thanks!

**Comment by SHWotever** [comment]
Hi ! This is the exact purpose of custom channels allocation, it allows you to add the same effect (with different settings) and map it to a specific set of shakers : https://github.com/SHWotever/SimHub/wiki/ShakeIt-V3-Effects-configuration#custom-channel-allocation

Nicolas

**Comment by nagler2** [comment]
Ok, mission completed. Thank you! ) I needed to write this email to understand how this beautiful program works. After several years using)


Thank you very much for quick response. I did not find the option I looked for. Look at the attached picture. For each effect I have one frequency, it does not matter how many shakers are there.


I tried to add second same effect (road vibration, for example) and I had this:




But there is no second “Road vibration”
